app/lib/features/news/widgets/news_side_bar.dart

```diff
@@ -197,7 +197,10 @@ class ActionBox extends ConsumerWidget {
             builder: (context) => RedactContentWidget(
               title: 'Remove this post',
               eventId: news.eventId().toString(),
-              onSuccess: () => ref.invalidate(newsListProvider),
+              onSuccess: () {
+                context.pop();
+                ref.invalidate(newsListProvider);
```

Comment:
not sure if this indeed works everytime...is it possible to include integration test for deleting update? That'll make us somewhat confident!

Reply:
News reduction bugs was already fixed on past [PR-Live redaction support for spaces.](https://github.com/acterglobal/a3/pull/1448).

In this PR I have just fixed minor UI glitches with  `context.pop();` to close deletion menu after news post deleted successfully.

And yes, Surely I will add end-to-end integration test for News Post Delete in separate PR. :)

Reply:
> is it possible to include integration test for deleting update? That'll make us somewhat confident!

Done. Added it here. :)
https://github.com/acterglobal/a3/pull/1463